README.md

```diff
@@ -1,18 +1,6 @@
 Chitter Challenge
```

Comment:
informative, easy to read and understand. Includes instructions on how the app works and how to setup the database.